File: app/src/at-baole/java/asiantech/internship/summer/restapi/model/ImageItem.java
package asiantech.internship.summer.restapi.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ImageItem {

    @SerializedName("image_id")
    @Expose
    private String imageId;
    @SerializedName("permalink_url")
    @Expose
    private String permalinkUrl;
    @SerializedName("thumb_url")
    @Expose
    private String thumbUrl;
    @SerializedName("url")
    @Expose
    private String url;
    @SerializedName("type")
    @Expose
    private String type;

    public String getImageId() {
        return imageId;
    }

    public void setImageId(String imageId) {
        this.imageId = imageId;
    }

    public String getPermalinkUrl() {
        return permalinkUrl;
    }

    public void setPermalinkUrl(String permalinkUrl) {
        this.permalinkUrl = permalinkUrl;
    }

    public String getThumbUrl() {
        return thumbUrl;
    }

    public void setThumbUrl(String thumbUrl) {
        this.thumbUrl = thumbUrl;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: app/src/at-hungtran/java/asiantech/internship/summer/kotlin/model/ImageItem.kt
package asiantech.internship.summer.kotlin.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ImageItem {
    @SerializedName("image_id")
    @Expose
    private var imageId: String? = null
    @SerializedName("permalink_url")
    @Expose
    private var permalinkUrl: String? = null
    @SerializedName("thumb_url")
    @Expose
    private var thumbUrl: String? = null
    @SerializedName("url")
    @Expose
    private var url: String? = null
    @SerializedName("type")
    @Expose
    private var type: String? = null

    fun getImageId(): String? {
        return imageId
    }

    fun setImageId(imageId: String) {
        this.imageId = imageId
    }

    fun getPermalinkUrl(): String? {
        return permalinkUrl
    }

    fun setPermalinkUrl(permalinkUrl: String) {
        this.permalinkUrl = permalinkUrl
    }

    fun getThumbUrl(): String? {
        return thumbUrl
    }

    fun setThumbUrl(thumbUrl: String) {
        this.thumbUrl = thumbUrl
    }

    fun getUrl(): String? {
        return url
    }

    fun setUrl(url: String) {
        this.url = url
    }

    fun getType(): String? {
        return type
    }

    fun setType(type: String) {
        this.type = type
    }
}